projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.model.ts

```diff
@@ -0,0 +1,7 @@
+import { Observable } from 'rxjs';
+import { MerchandisingProduct } from '../../model';
+
+export interface MerchandisingCarouselModel {
```

Comment:
Now that this model is used across 2 files I pulled it into its own file. I've left it in this dir as it is not used outside of here. Would it be preferred to have it in the model dir but not exported as part of the module?

Reply:
Preferred to have it in a model directory and have it exported in an index.ts, just for ease of convenience in the future for things that might use it